docs/html-css/week-1/lesson.md

```diff
@@ -22,15 +25,31 @@ Learning objectives for this lesson can be found [here](./learning-objectives.md
 
 ## What makes a web page?
 
-<img src={require('!file-loader!../assets/webpage-parts.png').default}/>
+<img src={require('!file-loader!../assets/webpage-parts.png').default}
+  alt="Diagram showing that structure (HTML), presentation (CSS), and behaviour (JavaScript) are interlinked"/>
 
-**Separation of Concerns:** In computer science, separation of concerns (SoC) is a design principle for separating a computer program into distinct sections, such that each section addresses a separate concern. (Wikipedia)
+A web page consists of three programming languages: HTML, CSS, and JavaScript.
+They define the structure (HTML), presentation (CSS), and behaviour (JavaScript) of the page.
+
+> **Separation of Concerns:** In computer science, separation of concerns (SoC) is a design principle for separating a computer program into distinct sections, such that each section addresses a separate concern. ([Wikipedia](https://en.wikipedia.org/wiki/Separation_of_concerns))
 
 > **Principles** - During the course, we will highlight principles that apply to the specific topic but also to Computer Science and Programming in general. It is important to understand these principles, terms and apply them to your thinking in general.
 
-## HTML Syntax
+## HTML anatomy
```

Comment:
HTML anatomy is added here so that students can refer to the syllabus to remember what the different parts of an HTML element are called.